### PR TITLE
#43 actions for add issue and pr to project.

### PR DESCRIPTION
**Pull request by xandercai**
DESCRIPTION OF PR:

## Developer Checklist
- [ x ] Make code change
- [ ] Update tests
  - [ ] Update / create new tests
  - [ ] Ensure these tests have the expected behaviour
  - [ ] Test locally and ensure tests are passing
- [ ] Update documentation
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

## Reviewer Checklist
- [ ] Check new code for code smells
- [ ] Check new tests
  - [ ] Ensure adequate coverage
  - [ ] Check for code smells within tests
- [ ] Check if documentation needs updating
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

### CODE REVIEW

File: .github/workflows/add-issue-and-pr-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  pull_request:
+    types:
+      - opened
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue and/or PR to project
```
Comment (LukeParky) [inline .github/workflows/add-issue-and-pr-to-project.yml:13]:
```suggestion
  add-to-project:
    name: Add issue to project
```

**Review by LukeParky — APPROVED** [review]
Good clear descriptions in this job. I added some suggestions because of the way we use the project. Once the action is successful this can be merged. Nice work on your first pull request @xandercai

**Comment by xandercai** [comment]
Changing token.

**Comment by xandercai** [comment]
To follow the workflow.

**Comment by caseyli92** [comment]
Nice work @xandercai. I approve the changes.

**Review by caseyli92 — APPROVED** [review]
Nice work @xandercai. I approve the changes.

**Review by LukeParky — APPROVED** [review]

**Pull request merged**